docs/interpreter/shell.md

```diff
@@ -82,3 +87,8 @@ export LAUNCH_KERBEROS_REFRESH_INTERVAL=4h
 # Change kinit number retries (default value is 5), which means if the kinit command fails for 5 retries consecutively it will close the interpreter. 
 export KINIT_FAIL_THRESHOLD=10
 ```
+
+## Object Interpolation
+The shell interpreter also supports interpolation of `ZeppelinContext` objects into the paragraph text.
+Details of this feature can be found in the Spark interpreter documentation under 
+[Object Interpolation](spark.html#object-interpolation)
```

Comment:
since this points to spark doc, perhaps add a quick example here with shell?

Reply:
actually, let's have to example in spark because..

Reply:
I have expanded and clarified the description of the feature here, and hope it is not confusing for readers now. Also explained that object interpolation is implemented per-interpreter, can be enabled/disabled independently per-interpreter, and has a different parameter name for enabling the feature in each interpreter. The parameter name is documented in the _Configuration_ part of both the implementing interpreters (SparkSQL, and Shell)